### ISSUE TITLE
Adding image support to the Graph2

### ISSUE THREAD

**Issue by waterkey15** [hpcc-systems/Visualization#4072]
We are using Graph2 of Visualization project of HPCC-Systems for our project. We want to use images more specifically country images instead of icons in the Graph2 Chart. Please see the below link for the project we are referring.

https://raw.githack.com/hpcc-systems/Visualization/trunk/apps/docs/index.html

We already have a folder which we store the country flags in png format but ideally it should be like <img> tags in HTML which allows us to use it with file in the project, link or source. 

Please see the below image. 

![images for hpcc graph](https://user-images.githubusercontent.com/54453298/221950795-9ca83105-73af-410a-b2f2-25a0542cf9b8.png)

Basically icons should be replaced by an image either from a link or by an .png .jpeg etc. (basically any kind of image) file
